src/openrct2/park/ParkFile.h

```diff
@@ -13,6 +13,9 @@ namespace OpenRCT2
     // The minimum version that is forwards compatible with the current version.
     constexpr uint32_t PARK_FILE_MIN_VERSION = 0x9;
 
+    // The minimum version that is backwards compatible with the current version.
+    constexpr uint32_t PARK_FILE_MIN_SUPPORTED_VERSION = 0x1;
```

Comment:
Did we start at 0x1, or at 0x0?

Reply:
0, but I can't use 0 because the compilers complain that I am doing 0 < 0.

Reply:
Sure, but this means that .park files that loaded correctly before will not load now. There has to be a way to satisfy the compilers without cutting off v0 support so quickly.

Reply:
Sounds like this check is useless for now, until we drop support for older versions.

Reply:
I think we could do two things:

- Leave out the check for "too old" parks, or
- Wrap the code in an `if (PARK_FILE_MIN_SUPPORTED_VERSION > 0)` to satisfy the compilers.

Reply:
I have only done this to satisfy @duncanspumpkin's request. Check with him.
I haven't actually tested whether v0 parks still load correctly, maybe they don't?

Reply:
I wouldn't expect v0 parks to load. There was a whole bunch of issues in like v3.